src/core/reducers/autocomplete.js

```diff
@@ -0,0 +1,69 @@
+export const AUTOCOMPLETE_LOADED = 'core/autocomplete/AUTOCOMPLETE_LOADED';
+export const AUTOCOMPLETE_STARTED = 'core/autocomplete/AUTOCOMPLETE_STARTED';
+export const AUTOCOMPLETE_ABORTED = 'core/autocomplete/AUTOCOMPLETE_ABORTED';
+
+const initialState = {
+  loading: false,
+  suggestions: [],
+};
+
+export function autocompleteStart({ errorHandlerId, filters }) {
+  if (!errorHandlerId) {
+    throw new Error('errorHandlerId is required');
+  }
+
+  if (!filters) {
+    throw new Error('filters are required');
+  }
+
+  return {
+    type: AUTOCOMPLETE_STARTED,
+    payload: { errorHandlerId, filters },
+  };
+}
+
+export function autocompleteAbort() {
+  return { type: AUTOCOMPLETE_ABORTED };
+}
+
+export function autocompleteLoad({ results }) {
+  if (!results) {
+    throw new Error('results are required');
+  }
+
+  return {
+    type: AUTOCOMPLETE_LOADED,
+    payload: { results },
+  };
+}
+
+export default function reducer(state = initialState, action = {}) {
+  const { payload } = action;
+
+  switch (action.type) {
+    case AUTOCOMPLETE_ABORTED:
```

Comment:
This is a nitpick but `AUTOCOMPLETE_CANCELLED` would be a bit nicer; it's a less harsh, programmer-y type word. Technically aborted is the wrong word anyway: it means stopped prematurely *because of a fault*; this action happens because a user stops/changes their search.

I'm not trying to be a dictionary nerd, but we use `STARTED` instead of `INITIALISED` most of the time, I think `CANCELLED` beats `ABORTED`.